sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/Internals/TraceHelper.cs

```diff
@@ -300,5 +300,55 @@ private static void AddTelemetryFromActivityEvents(Activity activity, TelemetryI
                 BaseData = exceptionData,
             };
         }
+
+        internal static void AddEnqueuedTimeToMeasurements(Activity activity, IDictionary<string, double> measurements)
+        {
+            if (activity.Links != null && activity.Links.Any())
+            {
+                if (TryGetAverageTimeInQueueForBatch(activity.Links, activity.StartTimeUtc, out long enqueuedTime))
+                {
+                    measurements["timeSinceEnqueued"] = enqueuedTime;
```

Comment:
is that a special key?

Reply:
Yes, this is the special key that UI looks up to show time in queue in E2E parameter.